Gemfile

```diff
@@ -6,17 +6,13 @@ git_source(:github) do |repo_name|
 end
 
 gem 'activemodel-serializers-xml'
-# gem 'bulma-rails', '~> 0.4.0'
-gem 'coveralls', require: false
 gem 'globalize'
 gem 'globalize-accessors'
 gem 'rails-i18n'
-gem 'sqlite3'
-gem 'bootstrap'
 gem 'pg'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 6.0.2.1'
+gem 'rails', '7.2.2.1'
```

Comment:
Just wanted to let you know that the following deprecation warning is currently shown in the browser console:

> DEPRECATION: action_cable.js has been renamed to actioncable.js – please update your reference before Rails 8

Reply:
I got the deprecation warning removed by replacing `require action_cable.js` by `require actioncable.js` in `app/assets/javascripts/cable.js`, see also this [post](https://stackoverflow.com/questions/75235925/how-to-fix-the-warning-deprecation-action-cable-js-has-been-renamed-to-action). At some point we will probably have to do this in MaMpf as well.